### PR TITLE
Support for Red Hat disabled rules via API rule_status parameter

### PR DESCRIPTION
**Pull request by mhuth**
Replaces PR https://github.com/RedHatInsights/insights-advisor-frontend/pull/755

Uses the yet-to-be-approved rule_status parameter instead of the reports_shown parameter, thus keeping backwards compatibility.  This PR will have to wait until the API PR https://github.com/RedHatInsights/insights-advisor-api/pull/645 goes in though.

Screenshots look much the same, main thing is the use of the new rule_status parameter:
rule_status_rhdisabled

### CODE REVIEW

**Review by AllenBW — COMMENTED** [review]
Nothin' jumps out at me, but will wait on the api work to make it in n poke around 👍 🤗

**Comment by terezanovotna** [comment]
Great iterations!

- the notification should be on the white background as in mocks [here](https://marvelapp.com/prototype/e5gcj47/screen/71274223)

Screen Shot 2020-09-24 at 18 30 44


Looking very good!

**Comment by mhuth** [comment]
Ok, got the alert in the correct place now and looks like the mocks.  You might want to check my CSS though coz it feels a bit hack-ish.
alert_now_like_the_mocks

**Review by terezanovotna — DISMISSED** [review]
👏👏👏

File: src/SmartComponents/Recs/List.scss [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
-    > * {
+    .pf-c-title {
         display:inline-block;
     }
     .pf-c-button {
         line-height: var(--pf-global--LineHeight--sm);
     }
+    .alertOverride {
+        font-size: var(--pf-global--FontSize--md);
+        margin-top: 24px;
```
Comment (LI0131) [inline src/SmartComponents/Recs/List.scss:12]: Hey @mhuth you could probably switch this px value for `margin-top: var(--pf-global--spacer--lg);`

Comment (mhuth) [reply src/SmartComponents/Recs/List.scss]: Done!  Good suggestion @LI0131 !

**Review by AllenBW — APPROVED** [review]
LETS DO IT
:godmode: 
🏆 for @mhuth

**Pull request merged**